config/locales/en.yml

```diff
@@ -35,3 +35,6 @@ en:
       registration: 'Registration'
       login: 'Login'
       logout: 'Logout'
+
+  user:
```

Comment:
use `active_record.yml` for messages like this one

Reply:
How can i get into it?